pkg/cmd/server/start.go

```diff
@@ -58,7 +58,7 @@ type PorchServerOptions struct {
 	RepoSyncFrequency                time.Duration
 	UseGitCaBundle                   bool
 	DisableValidatingAdmissionPolicy bool
-	MaxRequestBodySize               int64
+	MaxRequestBodySize               int
```

Comment:
Any reason this must be auto-selected at compile time?

Reply:
The only reason is that GRPC mas-size expects an `int`, the apiserver runtime max-request-size expects an `int64` and I considered safer to convert from int to int64, than the other way around.